### ISSUE TITLE
PhotoCards fail to render for some items.

### ISSUE THREAD

**Issue by DavidThePurple** [DavidThePurple/Inventorinator#13]
<img width="2559" height="1438" alt="Image" src="https://github.com/user-attachments/assets/3f100573-63b7-45d3-80bd-3f1e368a60d4" />

Relevance:  35bec1d
